cmd/down.go

```diff
@@ -56,5 +58,6 @@ var downCmd = &cobra.Command{
 func init() {
 	downCmd.Flags().BoolVar(&DownEmptyVols, "emptyvols", false, "Use Empty Volumes. Do not generate PVCs")
 	downCmd.Flags().IntVar(&DownReplicas, "replicas", 1, "Specify the number of repliaces in the generate resource spec")
+	downCmd.Flags().StringVar(&DownNamespace, "namespace", "", " Specify Namespace to deploy your application")
```

Comment:
There's a space before `Specify`

It may be good to add a default value to this as well, using `default` namespace.